File: monitor/v1/event/read-example-actorsid-resourcesid-error/read-example-actorsid-resourcesid-error.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Monitor.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var events = EventResource.Read(
            actorSid: "USd0afd67cddff4ec7cb0022771a203cb1",
            resourceSid: "PN4aa51b930717ea83c91971b86d99018f",
            limit: 20
        );

        foreach(var record in events)
        {
           Console.WriteLine(record.Sid);
        }
    }
}

File: monitor/v1/event/read-example-actorsid-resourcesid-error/read-example-actorsid-resourcesid-error.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

events = @client.monitor.events
                        .list(
                           actor_sid: 'USd0afd67cddff4ec7cb0022771a203cb1',
                           resource_sid: 'PN4aa51b930717ea83c91971b86d99018f'
                         )

events.each do |record|
  puts record.sid
end
